index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Landing page</title>
+    <link rel="stylesheet" href="./resources/css/style.css"/>
+</head>
+<body>
+    <nav>
+        <div class="heading">
+            <img class="logo"
+            src="./resources/images/36.jpg" alt="logo image"/>
+             <h2 class="nav_heading">FavCrunchies</h2>
+        </div>
+        <div class="right_cnt">
+        <ul>
+            <li><a href="/">Home</a></li>
+            <li><a href="/profile">Profile</a></li>
+            <li><a href="/project">Project</a></li>
+            <li><a href="/contact">Contact</a></li>
+            <li><a href="/about">About</a></li>
+        </ul>
+        <button class="btn">Sign up</button>
```

Comment:
This is also a child to the div it is not supposed to be on the same indentation